proto/stream_plan.proto

```diff
@@ -129,6 +137,8 @@ message BarrierMutation {
     ThrottleMutation throttle = 10;
     // Drop subscription on mv
     DropSubscriptionsMutation drop_subscriptions = 12;
+    // Alter sink with_option
+    AlterSinkConfigMutation alter_sink_config = 13;
```

Comment:
shall we make it `AlterConnectorPropsMutation`, it should contain a mapping from actor_id -> full setting for new connectors